scripts/build_existing_heating_distribution.py

```diff
@@ -82,7 +82,9 @@ def build_existing_heating():
     # distribute technologies to nodes by population
     pop_layout = pd.read_csv(snakemake.input.clustered_pop_layout, index_col=0)
 
-    nodal_heating = existing_heating.loc[pop_layout.ct]
+    nodal_heating = existing_heating.reindex(pop_layout.ct, fill_value=0).loc[
+        pop_layout.ct.unique()
+    ]
     nodal_heating.index = pop_layout.index
     nodal_heating = nodal_heating.multiply(pop_layout.fraction, axis=0)
```

Comment:
`
```suggestion
    nodal_heating = existing_heating.reindex(pop_layout.ct, fill_value=0)
    nodal_heating.index = pop_layout.index
    nodal_heating = nodal_heating.multiply(pop_layout.fraction, axis=0)
```

Reply:
`.loc[pop_layout.ct.unique()]` is a no-op.

Luckily. Since, if it wasn't, then `nodal_heating.index = pop_layout.index` would fail.